ch3/ex04.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
+# EX 04: create list A with 3 strings as elements, copy it to list B
+
+import random
+
+def main():
+	lista = []
+	for _ in xrange(3):
+		lista.append(random.choice('abcdefghijklmnopqrstuvwxyz'))
+	lista2 = lista[:]
+	print "this is the content of the original list: %s" % lista
```

Comment:
Please use the print function and .format instead of %, when you'll need to move to python 3 you'll be thankful